src/test/kotlin/com/atlassian/performance/tools/jiraactions/api/scenario/BackupConfigurationIT.kt

```diff
@@ -0,0 +1,168 @@
+
+class BackupConfigurationIT {
+    private val logger: Logger = LogManager.getLogger(this::class.java)
+
+    /**
+     * During the test, you can connect to WebDriver by the VNC viewer.
+     * DockerisedChrome opens the port 5900 so executing command
+     * docker ps
+     * will show you which port on your host is open and mapped to 5900 on the container.
+     * The default password is `secret`.
+     */
+    @Test
+    fun shouldDisableBackup() {
+        val version = System.getenv("JIRA_SOFTWARE_VERSION") ?: "8.0.0"
+        logger.info("Testing Jira $version")
+        val scenario = TestingScenario()
+        val metrics = mutableListOf<ActionMetric>()
+        val actionMeter = ActionMeter(
+            virtualUser = UUID.randomUUID(),
+            output = CollectionActionMetricOutput(metrics),
+            clock = Clock.systemUTC(),
+            w3cPerformanceTimeline = DisabledW3cPerformanceTimeline()
+        )
+        val user = User("admin", "admin")
+        val userMemory = object : UserMemory {
+            override fun recall(): User {
+                return user
+            }
+
+            override fun remember(memories: Collection<User>) {
+                throw Exception("not implemented")
+            }
+        }
+
+        JiraCoreFormula.Builder()
+            .version(version)
+            .build()
+            .provision()
+            .use { jira ->
+                DockerisedChrome().start().use { browser ->
+                    val webJira = WebJira(
+                        browser.driver,
+                        jira.getUri(),
+                        user.password
+                    )
+                    scenario.getLogInAction(
+                        webJira,
+                        actionMeter,
+                        userMemory
+                    ).run()
+                    scenario.getSetupAction(
+                        webJira,
+                        actionMeter
+                    ).run()
+
+                    goToServices(browser.driver, jira)
+
+                    val backupId = "del_10001"
+                    val elementPresent = browser.driver.isElementPresent(By.id(backupId))
+                    Assertions.assertThat(elementPresent).isFalse()
+                }
+            }
+    }
+
+    @Test
+    fun shouldDisableBackups() {
+        val version = System.getenv("JIRA_SOFTWARE_VERSION") ?: "8.0.0"
+        logger.info("Testing Jira $version")
+        val scenario = TestingScenario()
+        val metrics = mutableListOf<ActionMetric>()
+        val actionMeter = ActionMeter(
+            virtualUser = UUID.randomUUID(),
+            output = CollectionActionMetricOutput(metrics),
+            clock = Clock.systemUTC(),
+            w3cPerformanceTimeline = DisabledW3cPerformanceTimeline()
+        )
+        val user = User("admin", "admin")
+        val userMemory = object : UserMemory {
+            override fun recall(): User {
+                return user
```

Comment:
You can use the `Convert to expression body` IDE action on both `UserMemory` functions to the impl shorter.
`shouldDisableBackups` is very long and methods should fit on one screen entirely.